raiden/ui/console.py

```diff
@@ -216,27 +224,6 @@ def register_token(self, token_address_hex):
         self._raiden.register_channel_manager(channel_manager.address)
         return channel_manager
 
-    def ping(self, peer_address_hex, timeout=4):
```

Comment:
The user cannot choose the `nonce` value, if he does one of the following will happen:

- He will choose a `nonce` that has been used by the health check, in this case he will get a false acknowledged confirmation (because the message was acknowledged in the past)
- He will choose a new `nonce` and then the health check will get a false acknowledgment.

Either way is incorrect.

Why does the user need to use ping messages? If one wants to know if the other node is online he can just start the health check task and check the node state, without knowledge of the underlying implementation.